changelog/1532.doc.1.rst

```diff
@@ -0,0 +1,3 @@
+Added the |sphinx-notfound-page|_ extension to Sphinx_ so that the
+documentation now has a :wikipedia:`404 <HTTP_404>` page in the same
+style as the rest of the documentation.
```

Comment:
I kinda wonder whether these extensions need changelog entries, given they're not really user facing. Certainly does no harm to keep them, though!

Reply:
Agreed — I was wondering the same thing, and came to the same conclusion.  Thank you for the review!